pkgs/applications/misc/myanimelist-cli/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, curl
+, json_c
+, libbsd
+, myanimelistClientId ? null,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "myanimelist-cli";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "jmakhack";
+    repo = "myanimelist-cli";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-EmdkPpYEUIk9hr6rbnixjvznKSEnTCSMZz/17BfHGCk=";
+  };
+
+  buildInputs = [
+    curl
+    json_c
+    libbsd
+  ];
+
+  # Where to one: https://myanimelist.net/apiconfig, select the type "other".
+  # Alternatively, set the env var MYANIMELIST_CLIENT_ID
+  postPatch = lib.optionalString (myanimelistClientId != null) ''
+    substituteInPlace src/mya.c \
+      --replace \
+        '    "YOUR TOKEN HERE"' \
+        '    "'${lib.escapeShellArg myanimelistClientId}'"'
+  '';
```

Comment:
We should omit this. I'm not 100% sure how sensitive the client ID is, but the environment variable seems perfectly reasonable, no need to make the client ID world-readable in the nix store.

Reply:
I removed it, but moved the and rephrased the comment documenting the env variable to lie near the meta block.